packages/components/inputs/rich-text-utils/src/rich-text-body/dropdown.styles.tsx

```diff
@@ -41,11 +41,11 @@ const getButtonStyles = (props: TDropdownStylesProps) => [
     justify-content: center;
     align-items: center;
     padding: ${props.isStyleButton
-      ? `calc(${designTokens.spacing10} - 1px) ${designTokens.spacing20}`
-      : designTokens.spacing10};
+      ? `${designTokens.paddingForLocalizedRichTextDropdownButton} ${designTokens.spacing20}`
```

Comment:
Could we include the `spacing20` within the new token?
It seems the the new token only includes vertical padding and I guess it should include both.